index.html

```diff
@@ -83,16 +83,16 @@ <h2>Implementations</h2>
             the Swiss-Army chainsaw of all things IPFS.
           </li>
           <li>
-            <a href="https://helia.io/">Helia</a> (JS/TS): general-purpose JS implementation of IPFS.
+            <a href="https://helia.io/">Helia</a> (JS/TS): a browser- and CDN-friendly, modular, "import only what you need" JS implementation of IPFS.
           </li>
         </ul>
       </section>
       <section id="spec">
         <h2>Specification</h2>
         <p>
           There are two specifications in DASL: <a href="#cid">CIDs</a> and <a href="#dcbor42">dCBOR42</a>. CIDs
-          (Content IDs) are identifiers used for content addressing and dCBOR42 (deterministic CBOR with tag 42)
-          is a serialization format that is deterministic and supports CID links.
+          (Content IDs) are opaque identifiers used for addressing resources by their contents, as in IPFS; dCBOR42 (deterministically-serialized CBOR with optional CBOR tag <code>42</code> supported)
```

Comment:
```suggestion
          (Content IDs) are identifiers used for addressing resources by their contents, as in IPFS; dCBOR42 (deterministically-serialized CBOR with optional CBOR tag <code>42</code> supported)
```

They're not opaque! On the contrary, they have all the info you need, transparently.